certified-connectors/MoreApp Forms/apiDefinition.swagger.json

```diff
@@ -653,7 +676,16 @@
           "type": "string"
         },
         "data": {
-          "type": "object"
+          "type": "object",
+          "x-ms-dynamic-schema": {
+            "operationId": "GetFormVersion",
+            "value-path": "schema",
```

Comment:
Kindly check the below error and add description property 

##[error]The 'description' property is required. ValuePath: /paths/~1api~1v1.0~1forms~1customer~1:customerId:~1forms~1{formId}~1versions/get